pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -53,23 +61,26 @@
 
     private static final String TENANT = "tnx";
     private static final String NAMESPACE1 = TENANT + "/ns1";
+    private static final int NUM_BROKER = 1;
+    private static final int NUM_TC_PER_ = 1;
```

Comment:
is there a typo here ? why this variable ends with a '_' ?

Reply:
terribly sorry.
 I overlooked this detail. 
 Thanks for pointing out.